packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -111,13 +112,60 @@ contract L2ToL2CrossDomainMessengerTest is Test {
         assertEq(logs[0].topics[3], bytes32(messageNonce));
 
         // data
-        assertEq(logs[0].data, abi.encode(address(this), _message));
+        assertEq(logs[0].data, abi.encode(address(this), _message, address(0)));
 
         // Check that the message nonce has been incremented
         assertEq(l2ToL2CrossDomainMessenger.messageNonce(), messageNonce + 1);
     }
 
-    /// @dev Tests that the `sendMessage` function reverts when sending a ETH
+    /// @dev Tests that `sendMessage(uint256,address,bytes,address)` succeeds and emits the correct event.
```

Comment:
We could create an internal function for send message, and invoke it in 2 differerent tests: 1 with and 1 without entrypoint.
Even though this behavior is not very common, I think is ok to apply it here.

This is an example where they applied it:
https://github.dev/ethereum-optimism/optimism/blob/c29b2094a3cc6edceee061441654eb3a7ace145b/packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol#L213

Reply:
Ok, I think we are not going to add this change now.